pydis_site/apps/api/viewsets/bot/whitelist.py

```diff
@@ -0,0 +1,69 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import mixins
+from rest_framework.viewsets import GenericViewSet
+
+from pydis_site.apps.api.models.bot.whitelist import Whitelist
+from pydis_site.apps.api.serializers import WhitelistSerializer
+
+
+class WhitelistViewSet(
+    mixins.CreateModelMixin,
+    mixins.ListModelMixin,
+    mixins.DestroyModelMixin,
+    GenericViewSet,
```

Comment:
Wouldn't hurt to add `RetrieveModelMixin` and just make this CRUD. May come in handy in the future and it's a 1 line code change (well, besides the docstring) thanks to DRF's magic.